js/components/graph/Sidebar.js

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Focus from "./Focus";
+
+export default class Sidebar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      contentHidden: true,
+      focusDisabled: false,
+      focusActive: false,
+      graphActive: false,
+      graphName: "",
+      toggled: false
+    }
+  }
+
+  componentWillUpdate(prevProps) {
+    // Check to see if we have a graph from local storage on load
+    if (this.state.graphName.length == 0 && prevProps.graphName !== this.state.graphName) {
```

Comment:
Always use `===` for value equality, not `==`. (Check this throughout these files.)